sklearn/utils/estimator_checks.py

```diff
@@ -1360,6 +1390,62 @@ def check_classifiers_train(name, classifier_orig):
                 assert_array_equal(np.argsort(y_log_prob), np.argsort(y_prob))
 
 
+def check_outliers_train(name, estimator_orig):
+    X, _ = make_blobs(n_samples=300, random_state=0)
+    X = shuffle(X, random_state=7)
+    n_samples, n_features = X.shape
+    estimator = clone(estimator_orig)
+    set_random_state(estimator)
+
+    # fit
+    estimator.fit(X)
+    # with lists
+    estimator.fit(X.tolist())
+
+    y_pred = estimator.predict(X)
+    assert_equal(y_pred.shape, (n_samples,))
```

Comment:
also assert that it:
* is an array (though perhaps this is implied by `.shape`)
* has integer dtype
* has values in (-1, 1)

Avoid assert_equal in new tests. Use `assert` statements.